extensions.md

```diff
@@ -0,0 +1,59 @@
+# Registered Extensions
+
+The `extensions` field of a Cloud Event holds additional metadata in an event.
+This field may be both used for experimental features not yet ready for
+standardization or for fields that are domain-specific and not suitable for
+standardization.
+
+To encourage common design patterns where they make sense, an event source MAY
+choose to use a registered extension. When using an extension with a registered
+name, an event source SHOULD behave consistently with the description.
```

Comment:
s/SHOULD/MUST/ otherwise we don't get interop and lose the value.  In fact, you may want to add the boilerplate RFC2119 text to this doc too.

Reply:
@inlined ^^^